gradle-jdks/src/main/resources/gradlew-patch.sh

```diff
@@ -1,5 +1,5 @@
-
 # >>> Gradle JDK setup >>>
+# TODO(crogoz): what if we disable the gradle.jdk.setup.enabled=false?
 # !! Contents within this block are managed by 'palantir/gradle-jdks' !!
```

Comment:
Will add a FLUP - if an env variable is set (or if `gradle.jdk.setup.enabled=false`) -> we won't run the jdks-setup script.